import.py

```diff
@@ -540,16 +552,27 @@ def main() -> None:
     )
     args = parser.parse_args()
 
-    make_flags = args.make_flags + ["PERMUTER=1"]
+    settings: Mapping[str, object] = {}
+    settings_dir = find_root_dir(args.c_file, SETTINGS_FILES)
```

Comment:
Maybe add `["Makefile", "makefile", "build.ninja"]` to the list of searched files, and remove the find_root_dir calls in find_build_command_line? Feels a bit convoluted to search for root dir twice.